csharp/HOLMS.Platform/HOLMS.Platform/Types/Topics/FolioContextTopics.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace HOLMS.Platform.Types.Topics {
+    public class FolioContextTopics {
+        public static string ReservationsFolioClose = "folio.reservation_close";
```

Comment:
I'd prefer this be in the past tense. (a) "closed" is not ambiguous between "close", meaning "not far" vs. "close", the opposite of "open", and (b) I think the event indicates that it happened, not a request or something else?

Other than that, this looks fine.